packages/venia-concept/src/RootComponents/Product/Product.js

```diff
@@ -1,97 +1,137 @@
 import { Component, createElement } from 'react';
-import PropTypes from 'prop-types';
+import { shape, number, arrayOf, string } from 'prop-types';
 
 import classify from 'src/classify';
+import getUrlKey from 'src/util/getUrlKey';
 import Page from 'src/components/Page';
 import Carousel from 'src/components/ProductImageCarousel';
 import Options from 'src/components/ProductOptions';
 import Quantity from 'src/components/ProductQuantity';
 import RichText from 'src/components/RichText';
-import mockData from './mockData';
+import Currency from 'src/components/Currency';
 import defaultClasses from './product.css';
 
+import { mediaGalleryEntry, price } from 'src/shared/propShapes'
+
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+
+const productDetailQuery = gql`
+query productDetail($sku: String, $urlKey: String) {
+  productDetail: products(filter: { sku: {eq: $sku}, or: {url_key: {eq: $urlKey }}}, pageSize: 1, currentPage: 1) {
+    total_count
```

Comment:
Seems like multiple fields in this query aren't used in this PR afaict. Can we audit and remove unused fields?

Reply:
Sure!